docs/verify/simulate-with-qiskit-primitives.ipynb

```diff
@@ -0,0 +1,584 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "235b9def-4c99-4511-87dc-fe972acd1249",
+   "metadata": {},
+   "source": [
+    "# Exact simulation with Qiskit primitives"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eda9656b-3403-40e3-a466-6007466adb93",
+   "metadata": {},
+   "source": [
+    "The reference primitives in Qiskit perform local statevector simulations. These simulations do not support\n",
+    "modeling device noise, but are useful for quickly prototyping algorithms before looking into more advanced simulation\n",
+    "techniques (using Qiskit Aer) or running on real devices (Qiskit Runtime primitives).\n",
+    "\n",
+    "The `Estimator` primitive can compute expectation values of circuits, and the `Sampler` primitive can sample\n",
+    "from output distributions of circuits.\n",
+    "\n",
+    "The following sections show how to use the reference primitives to run your workflow locally."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "343d1fc6-f999-469c-9708-7536c3238b08",
+   "metadata": {},
+   "source": [
+    "## Use the reference `Estimator`\n",
+    "\n",
+    "There are two reference implementations of `Estimator` in `qiskit.primitives` that run on a local statevector\n",
+    "simulators: the [`StatevectorEstimator`](../api/qiskit/qiskit.primitives.StatevectorEstimator) class and the\n",
+    "[`Estimator`](../api/qiskit/qiskit.primitives.Estimator) class. The [`StatevectorEstimator`](../api/qiskit/qiskit.primitives.StatevectorEstimator)\n",
+    "implements the new Estimator V2 interface introduced in the Qiskit SDK 1.0, and offers additional input vectorization\n",
+    "features in comparison with the [`Estimator`](../api/qiskit/qiskit.primitives.Estimator) class, which implements the\n",
+    "legacy Estimator V1 interface. Both can take circuits, observables, and parameters as inputs and return the locally\n",
```

Comment:
Would it make sense to link to the migration guide when we talk about Estimator V1 interface?